stdlib/tkinter/ttk.pyi

```diff
@@ -961,9 +961,9 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         master: tkinter.Misc | None = None,
         *,
         class_: str = ...,
-        columns: str | list[str] | list[int] | list[str | int] | tuple[str | int, ...] = ...,
+        columns: _TreeviewColumnId | list[str] | list[int] | list[str | int] | tuple[str | int, ...] = ...,
```

Comment:
I think `int` does not make sense for `columns`. Why would you do `Treeview(columns=10)`? This will define one column with the name `10`, but someone will surely think it makes 10 different columns.

```suggestion
        columns: str | list[str] | list[int] | list[str | int] | tuple[str | int, ...] = ...,
```